.changeset/long-swans-search.md

```diff
@@ -0,0 +1,26 @@
+---
+'@manifest-ui/button': patch
```

Comment:
what a patch? not blocking just curious if we're adding new components, isn't that a feature (minor)